sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -6,15 +6,2076 @@
+        case_temp_high_alarm = self.xcvr_eeprom.read(consts.CASE_TEMP_HIGH_ALARM)
+        case_temp_low_alarm = self.xcvr_eeprom.read(consts.CASE_TEMP_LOW_ALARM)
+        case_temp_high_warn = self.xcvr_eeprom.read(consts.CASE_TEMP_HIGH_WARN)
+        case_temp_low_warn = self.xcvr_eeprom.read(consts.CASE_TEMP_LOW_WARN)
+        case_temp_dict = {'monitor value': case_temp,
+                          'high alarm': case_temp_high_alarm,
+                          'low alarm': case_temp_low_alarm,
+                          'high warn': case_temp_high_warn,
+                          'low warn': case_temp_low_warn}
+        return case_temp_dict
+    
+    def get_module_voltage(self):
+        '''
+        This function returns the monitored value of the 3.3-V supply voltage and its thresholds.
+        Unit in V
+        '''
+        voltage = self.xcvr_eeprom.read(consts.VOLTAGE)
+        voltage_high_alarm = self.xcvr_eeprom.read(consts.VOLTAGE_HIGH_ALARM)
+        voltage_low_alarm = self.xcvr_eeprom.read(consts.VOLTAGE_LOW_ALARM)
+        voltage_high_warn = self.xcvr_eeprom.read(consts.VOLTAGE_HIGH_WARN)
+        voltage_low_warn = self.xcvr_eeprom.read(consts.VOLTAGE_LOW_WARN)
+        voltage_dict = {'monitor value': voltage,
+                        'high alarm': voltage_high_alarm,
+                        'low alarm': voltage_low_alarm,
+                        'high warn': voltage_high_warn,
+                        'low warn': voltage_low_warn}
+        return voltage_dict
+
+    def get_txpower(self):
+        '''
+        This function returns the TX output power. Unit in mW
+        '''
+        tx_power = self.xcvr_eeprom.read(consts.TX_POW)
+        tx_power_high_alarm = self.xcvr_eeprom.read(consts.TX_POWER_HIGH_ALARM)
+        tx_power_low_alarm = self.xcvr_eeprom.read(consts.TX_POWER_LOW_ALARM)
+        tx_power_high_warn = self.xcvr_eeprom.read(consts.TX_POWER_HIGH_WARN)
+        tx_power_low_warn = self.xcvr_eeprom.read(consts.TX_POWER_LOW_WARN)
+        tx_power_dict = {'monitor value lane1': tx_power,
+                        'high alarm': tx_power_high_alarm,
+                        'low alarm': tx_power_low_alarm,
+                        'high warn': tx_power_high_warn,
+                        'low warn': tx_power_low_warn}        
+        return tx_power_dict
+
+    def get_rxpower(self):
+        '''
+        This function returns the RX input power. Unit in mW
+        '''
+        rx_power = self.xcvr_eeprom.read(consts.RX_POW)
+        rx_power_high_alarm = self.xcvr_eeprom.read(consts.RX_POWER_HIGH_ALARM)
+        rx_power_low_alarm = self.xcvr_eeprom.read(consts.RX_POWER_LOW_ALARM)
+        rx_power_high_warn = self.xcvr_eeprom.read(consts.RX_POWER_HIGH_WARN)
+        rx_power_low_warn = self.xcvr_eeprom.read(consts.RX_POWER_LOW_WARN)
+        rx_power_dict = {'monitor value lane1': rx_power,
+                        'high alarm': rx_power_high_alarm,
```

Comment:
can you rename the dict keys similar to the ones defined(get_rx_power() ) in sff8436?.